composer/utils/device.py

```diff
@@ -42,8 +42,13 @@ def get_device(device: Optional[Union[str, 'Device']]) -> 'Device':
                     'Unable to import torch_xla. Please follow installation instructions at https://github.com/pytorch/xla'
                 )
             device = DeviceTPU()
+        elif device.lower() == 'hpu':
+            if not is_hpu_installed():
+                raise ImportError('Unable to import habana-torch-plugin.')
+            import habana_frameworks
```

Comment:
why do we need to `import habana_frameworks` here?

Reply:
habana_frameworks loads habana executables required to interface with Pytorch. Since we deleted this module after importing it in "is_hpu_installed" call, therefore importing it again to ensure habana executables get loaded.